tests/pr_test.go

```diff
@@ -75,6 +79,85 @@ func setupOptionsQuickStartPattern(t *testing.T, prefix string, dir string) *tes
 	return options
 }
 
+type RecursiveCopy struct {
+	excludeDirs []string
+
+	includeFiletypes []string
+
+	includeDirs []string
+}
+
+func recursiveCopy(dir string, dirsToExclude []string, fileTypesToInclude []string) []string {
```

Comment:
Word "Copy" might be misleading, as we are not actually copying any files.

This function doesn't return an error, so it will be hard for the unit test to not continue (and fail) if this function encounters an error of any kind.